.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,5 @@
+**Description of the change**:
+
+**Reason for the change**:
+
```

Comment:
Can you please add a couple more things? @xndr7 noted, this is a really good example -
 you can copy/pasta it here if you would like! 
https://github.com/sendgrid/open-source-library-data-collector/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
#456 resolves this comment.